### ISSUE TITLE
Compatibility with Raspberry Pi

### ISSUE THREAD

**Issue by Tylersuard** [pytorch/pytorch#49784]
## 🚀 Feature
It would be great if we could just download and install Torch using pip install torch or pip install torch-gpu.

## Motivation

New models of Yolo are only written in PyTorch, which is great, I much prefer it to Tensorflow, but I would like to be able to use these fast new object detection models on my Raspberry Pi, like I can with older Yolo models.  As more efficient models are produced in PyTorch, it might be a good idea to make Torch more compatible with Raspberry Pi just to show off new Pytorch models' efficiency and accuracy on low-power devices.

## Pitch

I want Torch to be easily installed and used on the Raspberry Pi, particularly the Pi 4B.

## Alternatives

Tensorflow is awful, but it works on the Pi, so I have had to use it and downgrade to Yolov3.  Yolov5 is out but it is written in PyTorch, which is not compatible with the Pi.  I would like to use the latest software on my pi.  The latest software is written in PyTorch because it is amazing, but PyTorch is not compatible with the Pi at this time.

## Additional context
Please make it easily installable and usable.  pip3 install torch would be great.
